ReSharper.FSharp/test/data/features/quickFixes/removeUnusedNamedPattern/Unused record named pattern first position.fs

```diff
@@ -0,0 +1,5 @@
+type R = { F1: int; F2: int; F3: int }
+match { F1 = 1; F2 = 2; F3 = 3 } with
+| { F1 = f1{caret}; F2 = 2; F3 = 3 } -> ()
+| { F1 = 1; F2 = f2; F3 = 3 } -> ()
+| { F1 = f1; F2 = f2; F3 = f3 } -> ()
```

Comment:
Please remove the extra match branches: they don't add anything useful to the test while make it harder to see the important details from a quick glance.
```suggestion
type R = { F1: int; F2: int }
match { F1 = 1; F2 = 2 } with
| { F1 = f1{caret}; F2 = 2 } -> ()
```